frontend/src/components/opening.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Image from 'next/image';
+import { Opening } from '@/types';
+import 'pure-react-carousel/dist/react-carousel.es.css';
+import Link from 'next/link';
+import Toaster from '@/utils/toaster';
+import { Buildings } from '@phosphor-icons/react';
+import 'pure-react-carousel/dist/react-carousel.es.css';
+import { USER_PROFILE_PIC_URL } from '@/config/routes';
+import postHandler from '@/handlers/post_handler';
+import { SERVER_ERROR } from '@/config/errors';
+import { X } from '@phosphor-icons/react/dist/ssr';
+
+interface Props {
+    opening : Opening ;
+  setOpenings: React.Dispatch<React.SetStateAction<Opening[]>>;
+}
+
+const OpeningComponent = ({ opening, setOpenings }: Props) => {
+  const removeFlag = () => {
+    const toaster = Toaster.startLoad('Removing Flag', opening.id);
+    const URL = `/flags/openings/${opening.id}`;
+    postHandler(URL, {})
+      .then(res => {
+        if (res.statusCode === 200) {
+          setOpenings(prev => prev.filter(p => p.id != opening.id));
+          Toaster.stopLoad(toaster, 'Flag Removed', 1);
+        } else {
+          if (res.data.message) Toaster.stopLoad(toaster, res.data.message, 0);
+          else {
+            Toaster.stopLoad(toaster, SERVER_ERROR, 0);
+          }
+        }
+      })
+      .catch(err => {
+        Toaster.stopLoad(toaster, SERVER_ERROR, 0);
+      });
+  };
+  return (
```

Comment:
opening card should show details about the opening. (title, description, created_at, etc)